packages/flutter/lib/src/widgets/async.dart

```diff
@@ -302,7 +300,7 @@ class AsyncSnapshot<T> {
 ///    itself based on a snapshot from interacting with a [Stream].
 ///  * [FutureBuilder], which delegates to an [AsyncWidgetBuilder] to build
 ///    itself based on a snapshot from interacting with a [Future].
-typedef AsyncWidgetBuilder<T> = Widget Function(BuildContext context, AsyncSnapshot<T> snapshot);
+typedef AsyncWidgetBuilder<T extends Object> = Widget Function(BuildContext context, AsyncSnapshot<T> snapshot);
```

Comment:
This `extends Object` change is failing many google3 tests where generic types are omitted. Example code: https://gist.github.com/goderbauer/244c5f19f69bb36d4a136d176ad189c4